.erb-lint.yml

```diff
@@ -0,0 +1,69 @@
+---
+glob: "**/*.{html,text,js}{+*,}.erb"
+exclude:
+  - '**/app/components/status_card/view.html.erb'
```

Comment:
I couldn't figure out how to make this pass the linter 🤔

Reply:
I think if you enclose the href assigned to the a-tag in quotes it will pass:

`href="<%= target %>"`